package.json

```diff
@@ -147,6 +147,7 @@
     "slashify": "1.0.0",
     "sticky-sidebar": "3.3.1",
     "stream-mock": "2.0.5",
+    "timeago.js": "4.0.2",
```

Comment:
> Timeago is a jQuery plugin that makes it easy to support automatically updating fuzzy timestamps

Reply:
I think that might be a different package? [`timeago.js`](https://timeago.org/) has no jQuery dependency :)